plutus-use-cases/scripts/Main.hs

```diff
@@ -29,10 +28,8 @@ import Spec.TokenAccount qualified as TokenAccount
 import Spec.Vesting qualified as Vesting
 
 writeWhat :: Command -> String
-writeWhat (Scripts FullyAppliedValidators) = "scripts (fully applied)"
-writeWhat (Scripts UnappliedValidators)    = "scripts (unapplied)"
-writeWhat Transactions{}                   = "transactions"
-writeWhat MkTxLogs                         = "mkTx logs"
```

Comment:
Also I'm not sure if it's fine that I deleted that command. Maybe we should reorganize the collection of them since we are removing `ScriptValidationEvent`.